### PR TITLE
Sieve of eratosthenes in ruby

### PR DESCRIPTION
**Pull request by Basilleaf24**
Fixes #

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- 
- 

#### Files Added:
- 
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+end
+
+puts p.inspect
+
+#Output
+
+#Following are the prime numbers smaller than or equal to 30
+#2 3 5 7 11 13 17 19 23 29
+
+#
```
Comment (singh-shreya6) [inline Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.rb]: Please add a blank line at end of file

File: Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  next if o.include? i
+  ii = i * 2
+  while ii <= n.last do
+    o << ii
+    ii = ii + i
+
+  end
+#Print all the prime numbers less than or equal to the number
+  p << i unless o.include?(i)
+
```
Comment (singh-shreya6) [inline Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.rb]: One blank line is enough

File: Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
+#Enter the number below which you need to find prime numbers
+#In this case 2..30
+
+n = 2..30
```
Comment (singh-shreya6) [inline Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.rb]: If it possible to take dynamic inputs in Ruby, then please take the input n for Sieve dynamically.

**Comment by raghav-dalmia** [comment]
Are you working on it?

**Comment by singh-shreya6** [comment]
Are you working on this?

**Comment by singh-shreya6** [comment]
No activity for a long time so closing this PR. Please make a new one if you decide to do it again @Basilleaf24

**Pull request closed without merging**